Add Supply chain whitepaper directory structure and assessment (#611)
cncf/tag-security: supply-chain-security/supply-chain-security-whitepaper/README.md

> Software Supply Chain Security Whitepaper

Reviewer: ```suggestion
Software Supply Chain Security Paper
```

==========================================

# About

The Software Supply Chain Security Paper is a STAG effort 
to ensure the cloud native community has access to information about building, 
distributing, deploying, and running secure software supply chains.

## Updates to the paper 

The Software Supply Chain Security Paper (SSCSP) is intended to be a living 
document created and maintained for the community, by its members.  

Updates to the paper, suggestions for updates, or discussion for updates
should initiate with an [issue](https://github.com/cncf/sig-security/issues) submitted to the repo and labeled with

> "suggestion" and "whitepaper".

Reviewer: Agreed.  Call it "supply chain" @lumjjb

> "suggestion" and "whitepaper".

Reviewer: ```suggestion
"suggestion" and "supplychain".
```

### Markdown 

The living SSCSP is maintained in [markdown](https://github.com/cncf/sig-security/blob/master/supply-chain-security/software-supply-chain-security-whitepaper.md) and all updates will be made in
 markdown.  

### Contributing updates 

All members of the community are welcome to contribute updates to the SSCSP.
We ask potential contributors to refer to the original design decisions, 
listed below, as guidance when determining the content of their updates.

It is highly recommended that you seek peer review for your updates beyond that
of the Technical Leads and Co-Chairs of the STAG.

Once the PR is submitted, please place the link in the CNCF SIG-Security Channel
for the SSCSP: [#sig-security-supply-chain-wg](https://cloud-native.slack.com/archives/C01KL0B4LKC) to request a review.

### Versioning and publishing 

It is expected that many minor updates will occur, corrections to grammar, 
spelling, clarification in language, translations, etc.  When these occur 
they are considered minor changes to the overall content and will not warrant
 the regeneration of the PDF.

When significant changes to the intent, content, or numerous minor changes

> occur, the SSCSP working group will assess and determine if a new major version

Reviewer: I think governance docs are a larger discussion for papers going forward and should definitely be added.  #621

Software Supply Chain Security Whitepaper
==========================================

# About

> The Software Supply Chain Security Whitepaper is a SIG-Security effort 

Reviewer: ```suggestion
The Software Supply Chain Security Paper is a STAG effort 
```

to ensure the cloud native community has access to information about building, 
distributing, deploying, and running secure software supply chains.

## Updates to the paper 

> The Software Supply Chain Security Whitepaper (SSCSWP) is intended to be a living 

Reviewer: ```suggestion
The Software Supply Chain Security Paper (SSCSWP) is intended to be a living 
```

document created and maintained for the community, by its members.  

> Updates to the whitepaper, suggestions for updates, or discussion for updates

Reviewer: ```suggestion
Updates to the paper, suggestions for updates, or discussion for updates
```

should initiate with an [issue](https://github.com/cncf/sig-security/issues) submitted to the repo and labeled with
"suggestion" and "whitepaper".

### Markdown 

> The living SSCSWP is maintained in [markdown](https://github.com/cncf/sig-security/blob/master/supply-chain-security/software-supply-chain-security-whitepaper.md) and all updates will be made in

Reviewer: ```suggestion
The living SSCSP is maintained in [markdown](https://github.com/cncf/sig-security/blob/master/supply-chain-security/software-supply-chain-security-whitepaper.md) and all updates will be made in
```

 markdown.  

### Contributing updates 

> All members of the community are welcome to contribute updates to the SSCSWP.

Reviewer: ```suggestion
All members of the community are welcome to contribute updates to the SSCSP.
```

We ask potential contributors to refer to the original design decisions, 
listed below, as guidance when determining the content of their updates.

It is highly recommended that you seek peer review for your updates beyond that

> of the Technical Leads and Co-Chairs of the SIG.

Reviewer: ```suggestion
of the Technical Leads and Co-Chairs of the STAG.
```

Once the PR is submitted, please place the link in the CNCF SIG-Security Channel

> for the SSCSWP: [#sig-security-supply-chain-wg](https://cloud-native.slack.com/archives/C01KL0B4LKC) to request a review.

Reviewer: ```suggestion
for the SSCSP: [#sig-security-supply-chain-wg](https://cloud-native.slack.com/archives/C01KL0B4LKC) to request a review.
```

### Versioning and publishing 

It is expected that many minor updates will occur, corrections to grammar, 
spelling, clarification in language, translations, etc.  When these occur 
they are considered minor changes to the overall content and will not warrant
 the regeneration of the PDF.

When significant changes to the intent, content, or numerous minor changes

> occur, the SSCSWP working group will assess and determine if a new major version

Reviewer: ```suggestion
occur, the SSCSP working group will assess and determine if a new major version
```

of the PDF needs published.  When this decision is made, the markdown content
will be converted to text document and sent to the CNCF technical writers to
create the PDF.  The PDF will then be published back into the repository
annotating the new version, updating the links in the README.md accordingly.

Minor updates to the markdown shall receive a minor version bump indicated in the
Metadata table of the document and recorded as WIP.  When enough significant
changes have been recorded, the markdown will be placed "In Review" (via PR) and

> solicited to the CNCF SIG-Security and TOC mailing list for review, at a minimum.

Reviewer: ```suggestion
solicited to the CNCF STAG and TOC mailing list for review, at a minimum.
```

> Upon completion of review, the SIG-Security TOC Liaison shall provide final

Reviewer: ```suggestion
Upon completion of review, the STAG TOC Liaison shall provide final
```
